styleguide/components/Scaffold/Scaffold.css

```diff
@@ -66,6 +66,28 @@
   flex-basis: 100%;
 }
 
+.blockquote {
+  margin: 0;
+  padding: var(--size-regular);
+  border-left: 0.25rem solid var(--color-greyDark);
```

Comment:
Specify the border value in `px` too, as I don't think it needs to increase in size with the font-size. None of the other borders do 👍